engine/src/main/resources/assets/schemas/schemaCollisionMesh.json

```diff
@@ -0,0 +1,116 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "Physical configuration of a collision mesh, used for in-game collision physics.",
+  "type": "object",
+  "properties": {
+    "rigidBodies": {
+      "description": "Model representing the collision bounds of an object. Intended to be exported from utility applications",
+      "type": "array",
+      "items": {
+        "properties": {
+          "name": {
+            "type": "string",
+            "description": "Identifying id of this collision mesh.",
+            "pattern": "^\\w+:\\w+$"
+          },
+          "origin": {
+            "description": "The center reference point for the mesh.",
+            "type": "object",
+            "required": [
+              "x",
+              "y"
+            ],
+            "properties": {
+              "x": {
+                "type": "number"
+              },
+              "y": {
+                "type": "number"
+              }
+            }
+          },
+          "polygons": {
+            "type": "array",
+            "items": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "required": [
+                  "x",
+                  "y"
+                ],
+                "properties": {
+                  "x": {
+                    "type": "number"
+                  },
+                  "y": {
+                    "type": "number"
+                  }
+                }
+              }
+            }
+          },
+          "shapes": {
+            "type": "array",
+            "items": {
+              "type": "object",
+              "properties": {
+                "type": {
+                  "type": "string"
+                },
+                "vertices": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "required": [
+                      "x",
+                      "y"
+                    ],
+                    "properties": {
+                      "x": {
+                        "type": "number"
+                      },
+                      "y": {
+                        "type": "number"
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "circles": {
+            "type": "array",
+            "items": {
+              "type": "object",
+              "properties": {
+                "cx": {
+                  "type": "number",
+                  "properties": {
+                    "cx": {
+                      "type": "number"
+                    },
+                    "cy": {
```

Comment:
To clarify my last "something is fishy" comment, why is cy nested inside the properties of cx?

Reply:
I knew I missed something :/
Fixed to remove the unnecessary cx statement.